### DISCUSSION TITLE
UPLOAD - request for help!

### DISCUSSION THREAD

**Discussion by darbud1** [rejetto/hfs#427] [General]
Hello and welcome ... For years, I have been using HFS version 2.4.0 RC6 at school to work with students, which has not been developed for a long time. Recently, I was very happy to hear that the project was being developed again in a new version. Unfortunately, after downloading version 0.51.0-beta8, despite many attempts, I cannot configure it so that my students can upload their files to the server. Please explain or provide simple instructions on how to UPLOAD this version of HFS? Best regards and I wish you perseverance in developing this great tool Darek

**Comment by rejetto** [comment]
Hello Darek,
1. have a folder in the Virtual File System
2. be sure it is bound to a folder on disk (the field is called Load content from disk)
3. click on "Who can upload", and decide the policy you desire.
4. from now on, users with permission (decided by you on step 3) will see a button "upload" when they access such folder

**Comment by darbud1** [comment]
Thank you very much for your quick reply. 
Tomorrow at school I will test it with my students. 
Best regards, rejetto :-))

**Comment by rejetto** [comment]
happy to help!
i just created a page on the wiki
https://github.com/rejetto/hfs/wiki/Upload
feel free to enhance it or comment

**Comment by rejetto** [comment]
I'd be glad to hear how you use HFS with your students.
And if it's possible also to know in what country is this school.

**Comment by darbud1** [comment]
The school is in Poland and I use HFS to share educational materials during lessons with students and to collect work and tasks from them in electronic form :-) It is a fast, safe way in our school local network

**Comment by darbud1** [comment]
I managed to UPLOAD and everything works. 
Thank you again for your help. 
Will it be possible to change the language of the admin panel in the near future???

**Comment by rejetto** [comment]
it's not in my plans, there's a ton of work to do before that.
i'm glad that it worked!
